web/api/queries/max_s/max_s.c

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "max_s.h"
+
+// ----------------------------------------------------------------------------
+// max_s
+
+struct stats_max {
+    calculated_number max;
+    size_t count;
+};
+
+void *stats_create_max(RRDR *r) {
+    (void)r;
```

Comment:
We have a macro called [`unused`](https://github.com/netdata/netdata/blob/7f39ccb6d3ca7e45745c3e690eeeaa9434356d8b/libnetdata/libnetdata.h#L301) that does exactly what you are writing here, please, use it for we keep an unique pattern.